shell/edit/logging.banzaicloud.io.output/index.vue

```diff
@@ -54,33 +59,56 @@ export default {
       return !isEmpty(correctedSpecProvider) && !isEqual(correctedSpecProvider, provider.default);
     });
 
-    const selectedProvider = selectedProviders?.[0]?.value || providers[0].value;
+    const hasMultipleProvidersSelected = selectedProviders?.length > 1;
 
-    let bufferYaml;
-
-    if ( !isEmpty(this.value.spec[selectedProvider]?.buffer) ) {
-      bufferYaml = jsyaml.dump(this.value.spec[selectedProvider].buffer);
-    } else {
-      bufferYaml = createYaml(schemas, `logging.banzaicloud.io.v1beta1.output.spec.${ selectedProvider }.buffer`, []);
-      // createYaml doesn't support passing reference types (array, map) as the first type. As such
-      // I'm manipulating the output since I'm not sure it's something we want to actually support
-      // seeing as it's really createResourceYaml and this here is a gray area between spoofed types
-      // and just a field within a spec.
-      bufferYaml = bufferYaml.substring(bufferYaml.indexOf('\n') + 1).replace(/# {2}/g, '#');
-    }
+    const selectedProvider = selectedProviders?.[0]?.value || providers[0].value;
 
     return {
-      bufferYaml,
-      initialBufferYaml:            bufferYaml,
+      schemaDefinition: null,
+      bufferYaml:       '',
       providers,
       selectedProvider,
-      hasMultipleProvidersSelected: selectedProviders?.length > 1,
-      selectedProviders,
+      hasMultipleProvidersSelected,
       LOGGING
     };
   },
 
   computed: {
+    schemas() {
```

Comment:
Had a thought. Given that we have an async fetch which might take a while to complete, think we'll need to
1. add  
   ```
   <Loading v-if="$fetchState.pending" />
   <div v-else class="output">
   ```
2. Then initialBufferYaml contents can just go after the `await resourceSchema.fetchResourceFields();` in fetch

Reply:
Good idea. Works like a charm. Updated